db/migrate/20181025230931_add_comments_to_vms_table.rb

```diff
@@ -0,0 +1,102 @@
+    change_column_comment :vms, :cloud_network_id, "ID of the cloud (virtual) network the VM/Template is associated with."
+    change_column_comment :vms, :cloud_subnet_id, "Subnet ID associated with the VM."
+    change_column_comment :vms, :cloud_tenant_id, "ID of the cloud tenant associated with the VM."
+    change_column_comment :vms, :config_xml, "The VM or Template configuration in XML format. Deprecated."
+    change_column_comment :vms, :connection_state, "Indicates the connection state of the VM, e.g. connected, disconnected, etc."
+    change_column_comment :vms, :cpu_affinity, "A comma separated list of host CPUs to schedule the VM on. VMware only."
+    change_column_comment :vms, :cpu_hot_add_enabled, "Indicates if CPUs can be added to the VM while it is powered on."
+    change_column_comment :vms, :cpu_hot_remove_enabled, "Indicates if a CPU can be removed from a VM while powered on."
+    change_column_comment :vms, :cpu_limit, "The CPU utilization limit for the VM. VMWare only."
+    change_column_comment :vms, :cpu_reserve, "Amount of CPU that is guaranteed available for the VM. VMWare only."
+
+    change_column_comment :vms, :cpu_reserve_expand, "Amount the CPU can grow beyond its normal limits if there "\
+                                                     "are unreserved resources available. VMWare only."
+
+    change_column_comment :vms, :cpu_shares, "The number of CPU shares allocated. Used to determine resource allocation "\
+                                             "in case of resource contention. VMWare only."
+
+    change_column_comment :vms, :cpu_shares_level, "CPU shares allocation level. VMWare only."
+    change_column_comment :vms, :created_on, "The timestamp the VM was added to the app inventory."
+    change_column_comment :vms, :deprecated, "Indicates whether or not a template is deprecated. Google only."
+    change_column_comment :vms, :description, "A description of the VM that is typically set by the provider, but may also be edited."
+    change_column_comment :vms, :ems_created_on, "Date the associated EMS was created."
+    change_column_comment :vms, :ems_id, "ID of the ExtManagementSystem associated with the VM."
+    change_column_comment :vms, :evm_owner_id, "ID of VM owner from product database."
+    change_column_comment :vms, :ems_ref, "A native unique reference for the VM within the EMS."
+    change_column_comment :vms, :ems_ref_obj, "A YAML serialized version of the ems_ref."
+    change_column_comment :vms, :ems_cluster_id, "ID of associated cluster."
+
+    change_column_comment :vms, :fault_tolerance, "Indicates whether or not the VM is fault tolerant, i.e. if one host "\
+                                                  "crashes it will immediately pick up on another host in the cluster."
+
+    change_column_comment :vms, :flavor_id, "ID of associated flavor."
+    change_column_comment :vms, :format, "The format of the VM's disk, such as VMDK."
```

Comment:
Not sure if this is lower case or not, but the example needs to be accurate.

Reply:
@Fryguy I'm guessing it would depend on the provider. I don't remember where I got that example from now, maybe @mfeifer.

Reply:
I don't see this being set by any parser either, @roliveri do you know if this is set by smartstate?

Reply:
Confirmed by @gmcculloug to be deprecated.